rootfs/bin/is_running

```diff
@@ -4,4 +4,4 @@
 set -e
 
 # check if database is running
-gosu postgres pg_ctl status
+gosu postgres psql -l -t >/dev/null 2>&1
```

Comment:
I agree that calling `psql -l -t` is a reasonable solution *in the normal mode*.
With that said, I suspect that psql in rediness checks crashes psql behind WAL-E in the recovery mode..
I’m not sure the mechanism for now. But it is reduced crashes in the recovery mode by avoiding `psql` calls. At least my experience on my deis/database.